pkg/features/kube_features.go

```diff
@@ -261,6 +261,12 @@ const (
 	//
 	// Enables the regional PD feature on GCE.
 	GCERegionalPersistentDisk utilfeature.Feature = "GCERegionalPersistentDisk"
+
+	// owner: @krmayankk
+	// alpha: v1.10
+	//
+	// Disable runAsGroup
```

Comment:
nit: Rephrase to `Enables control over the primary group ID of containers' init processes.`